flow/include/flow/EncryptUtils.h

```diff
@@ -37,14 +37,19 @@
 
 #define SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_ID -2
 #define ENCRYPT_HEADER_DOMAIN_ID -3
-
-const std::string FDB_DEFAULT_ENCRYPT_DOMAIN_NAME = "FdbDefaultEncryptDomain";
+#define FDB_DEFAULT_ENCRYPT_DOMAIN_ID -4
 
 using EncryptCipherDomainId = int64_t;
 using EncryptCipherDomainNameRef = StringRef;
+using EncryptCipherDomainName = Standalone<EncryptCipherDomainNameRef>;
 using EncryptCipherBaseKeyId = uint64_t;
 using EncryptCipherRandomSalt = uint64_t;
 
+const EncryptCipherDomainNameRef FDB_DEFAULT_ENCRYPT_DOMAIN_NAME_REF =
```

Comment:
nit: could FDB_DEFAULT_ENCRYPT_DOMAIN_NAME_REF be ref of FDB_DEFAULT_ENCRYPT_DOMAIN_NAME?

Reply:
I think we can just remove FDB_DEFAULT_ENCRYPT_DOMAIN_NAME as it is not used anywhere.

Reply:
Better would be a hybrid; keep the name `FDB_DEFAULT_ENCRYPT_DOMAIN_NAME` but have it be a `EncryptCipherDomainNameRef`.

Reply:
I prefer explicit naming, especially the constants, will keep it unless someone feels strong about it :)